utils/check_config_attributes.py

```diff
@@ -271,6 +271,7 @@
         "attention_chunk_size",
     ],
     "Llama4VisionConfig": ["multi_modal_projector_bias", "norm_eps"],
+    "SmolLM3Config": ["no_rope_layer_interval"],
```

Comment:
llama4-style. not sure if we should continue the pattern or leave just the `no_rope_layers`